### PR TITLE
url bugs fixed

### PR DESCRIPTION
**Pull request by ChsPranav**
@Sudhee-bsp Made sure that Clip-ID URL will only accept Alphanumeric, otherwise it won't procced for creating a new clipboard

- Added Alphanumeric Feature in clipboard

### CODE REVIEW

File: src/components/board.jsx [hunk 1]
```diff
@@ -21,6 +21,7 @@ function Board() {
     <div className="container mt-4">
       {/* <h3 className="text_center">CLIPNOW</h3> */}
       <h5 className="text_center">Create your CLIP!</h5>
+      <p>Note: Only Alphanumerics are allowed</p>
```
Comment (Sudhee-bsp) [inline src/components/board.jsx:24]: @ChsPranav, Please remove the 'p' tag statement, as the ID validation message will be shown by the browser itself.

**Review by Sudhee-bsp — CHANGES_REQUESTED** [review]
Please fix with requested changes. @ChsPranav

**Comment by ChsPranav** [comment]
@Sudhee-bsp  changes are done as per your request. please check it out.

**Comment by Sudhee-bsp** [comment]
Reopening #11

**Review by Sudhee-bsp — APPROVED** [review]
Done.

**Pull request merged**